wagtailnhsukfrontend/templates/wagtailnhsukfrontend/image.html

```diff
@@ -0,0 +1,37 @@
+{% load wagtailimages_tags %}
+
+{% image value.content_image original as org_image %}
+{% image value.content_image width-320 as one_image %}
+{% image value.content_image width-375 as two_image %}
+{% image value.content_image width-414 as three_image %}
+{% image value.content_image width-768 as four_image %}
+{% image value.content_image width-1024 as five_image %}
+{% image value.content_image width-1280 as six_image %}
+<figure class="nhsuk-image">
+  <img
+    class="nhsuk-image__img"
+    src="{{ org_image.url }}"
+    sizes="
+        (max-width: 320px) 320px,
+        (min-width: 320px) and (max-width: 375px) 375px,
```

Comment:
Each entry in `sizes` should be `media-condition size` where
- media-condition = CSS media condition
- size = The width of the slot the image will fill when the media condition is true.

So for a large desktop matching the `(min-width: 1024px) and (max-width: 1280px)` media query, we don't expect the image size to be 1280px.

The media queries here should match the breakpoints in the frontend library. I think we need to have a chat with the frontend library devs and get some guidance in the service-manual about this.

Reply:
I've created an issue on the frontend library https://github.com/nhsuk/nhsuk-frontend/issues/422